frontend/src/components/link.js

```diff
@@ -3,6 +3,16 @@ import { Link } from 'react-router';
 import shortUUID from '../helpers/uuid'
 
 export class NomadLink extends Component {
+
+    findNodeNameById(nodeId) {
+        let r = Object.keys(this.props.nodeList).filter(_ => { return this.props.nodeList[_]['ID'] === nodeId; })
```

Comment:
I copied `_` from some other code in nomad-ui :P didn't even know it was possible until i saw it, i prefer a proper name ;)